src/core/common/email.ts

```diff
@@ -0,0 +1,5 @@
+export const extractDomain = (email: string): string => {
+  const domain = email.split("@")[1];
```

Comment:
This is the logic we use according to https://github.com/coralproject/talk/blob/develop/src/core/server/graph/schema/schema.graphql#L1799

Reply:
yeah, there are proper ways to validate (we use Joi's built in `.email(...)` validator to do full IANA checking... but that's all server side and we don't want to include that package client side right now (it's huge).

Reply:
good catch!